### PR TITLE
[RW-6636][risk=no] updated code for full_text and synonyms columns, changed parent_ids

### PR DESCRIPTION
**Pull request by jscherdin**
updated code for full_text and synonyms columns, changed some parent_ids = -1 for consistency

### CODE REVIEW

File: api/db-cdr/generate-cdr/make-bq-criteria-tables.sh [hunk 1]
```diff
@@ -1412,7 +1412,7 @@ bq --quiet --project_id=$BQ_PROJECT query --nouse_legacy_sql \
     )
 SELECT
     (SELECT MAX(id) FROM \`$BQ_PROJECT.$BQ_DATASET.cb_criteria\`)+1 AS id
-    , 0
+    , -1
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/make-bq-criteria-tables.sh:1415]: So changing these parentIds from 0 to -1 will break API calls that expect the parent id to be 0. I attached a screenshot of an example UI call that is using parentId

Screen Shot 2021-04-22 at 1 24 01 PM

**Review by freemabd — COMMENTED** [review]
Added a comment about the -1 parentId changes.

**Review by freemabd — APPROVED** [review]
So everything looks good. Approving barring the merge of Neha's PR.

File: api/db-cdr/generate-cdr/make-bq-criteria-tables.sh (hunk 1 above)
Comment (dolbeew) [reply api/db-cdr/generate-cdr/make-bq-criteria-tables.sh:1415]: That is correct, should be okay once this PR goes through: https://github.com/all-of-us/workbench/pull/4856

**Pull request merged**